CHANGES

```diff
@@ -21,6 +20,8 @@ arpack-ng - 3.8.0
 
 [ Kyle Guinn ]
  * [BUG FIX]: fix 'eval: Syntax error: "(" unexpected' error at build time.
+ * Only build shared libraries by default.
```

Comment:
Please add the argument to explain how to build static again :)

Reply:
Done.